dice.py

```diff
@@ -1,23 +1,88 @@
+"""
+A script to roll nDm where n is the number of die and m is how many faces each die has. The value of each die as well as
+the total is printed each time the script is run. 4k ultra HD graphics of the dice are also displayed.
+
+Current limitations:
+* Each die needs to have the same number of sides
+* The graphics only display if ALL dice have a value less than or equal to 6
+* The number of dice and how many faces they have is currently hardcoded under configurables
+* The return_dice_as_text function returns "a 11" instead of "an 11". I really CBA to fix that
+"""
+
 import random
 
-repeat = True
-while repeat:
-    no = random.randint(1, 6)
-    print("You rolled", no)
-    if no == 1:
-        print("You feel somewhat disappointed")
-    if no == 2:
-        print("You look at the two dots and begin to mourn a lover lost at sea")
-    if no == 3:
-        print("Three is the magic number, you turn into a frog")
-    if no == 4:
-        print("4! You win...not really, you're just rolling a dice. Get some mates!")
-    if no == 5:
-        print("You bleed a little")
-    if no == 6:
-        print("You begin to wonder what this point of this is")
-    if no == 7:
-        print("err...what? This is literally impossible")
-
-    print("Do you want to roll again Y/N (please roll again, it...it pleases me?)")
-    repeat = "Y" in input()
\ No newline at end of file
+"""Configurables"""
+number_of_dice = 3
+faces = 6
+
+
+def main():
+    repeat = True
+    while repeat:
+        display(roll(number_of_dice, faces))
+        print("Do you want to roll again Y/N (please roll again, it...it pleases me?)")
+        repeat = "Y" in input().upper()
+
+
+def roll(number_of_dice, faces):
+    return [random.randint(1, faces) for n in range(number_of_dice)]
```

Comment:
@barneyh89 Are you familiar with list comprehension? It's a very powerful tool but takes some time to wrap your head around